src/main/kotlin/no/nav/syfo/client/pdfgen/PdfGenClient.kt

```diff
@@ -80,7 +85,8 @@ class PdfGenClient(
 
     companion object {
         private const val API_BASE_PATH = "/api/v1/genpdf/isbehandlerdialog"
-        const val FORESPORSEL_OM_PASIENT_PATH = "$API_BASE_PATH/foresporselompasient" // TODO: Noe annet for påminnelse?
+        const val FORESPORSEL_OM_PASIENT_PATH = "$API_BASE_PATH/foresporselompasient"
+        const val FORESPORSEL_OM_PASIENT_PAMINNELSE_PATH = "$API_BASE_PATH/foresporselompasient_paminnelse"
```

Comment:
Jeg tror standarden i REST-endepunkter er bindestrek i stedet for understrek i urlen: https://stackoverflow.com/a/18450653
Så det går an å endre, med mindre noen har sterke meninger om noe annet 🤷🏼‍♂️

Reply:
Sant det! Nå tror jeg rest-endepunktet bare får navnet ut i fra navnet til templaten i [isdialogmelding](https://github.com/navikt/dialogmeldingpdfgen/tree/master/templates/isbehandlerdialog) og det blir kanskje litt rart å bruke bindestrek i det filnavnet? Ellers kan jeg godt rename den hvis vi tenker det er bedre.